tests/online/badssl.c

```diff
@@ -36,3 +36,12 @@ void test_online_badssl__self_signed(void)
 	cl_git_fail_with(GIT_ECERTIFICATE,
 			 git_clone(&g_repo, "https://self-signed.badssl.com/fake.git", "./fake", NULL));
 }
+
+void test_online_badssl__old_cipher(void)
+{
+	if (!g_has_ssl)
+		cl_skip();
+
+	cl_git_fail_with(GIT_ERROR,
+			 git_clone(&g_repo, "https://rc4.badssl.com/fake.git", "./fake", NULL));
```

Comment:
Not really happy about the error type here, but there doesn't seem to be an easy way to get a more specific error.